prep-html-skeleton/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <title>HTML Skeleton Exercise</title>
+</head>
+
+<body>
+  <h1>I built a proper HTML skeleton!</h1>
+  <script src="main.js"></script>
```

Comment:
You can leave out the script tag since we didn't make any JavaScript files for this exercise

Reply:
![Screen Shot 2021-09-18 at 2 14 56 PM](https://user-images.githubusercontent.com/72685552/133908720-e166317a-90ba-4348-947a-dff9edaf5dea.png)